core/log.h

```diff
@@ -101,6 +102,7 @@ class LogDetailed
                 std::cout << "|Debug] ";
                 break;
             case LogLevel::Info:
+                std::cout << ANSI_COLOR_BLUE;
```

Comment:
This is really not critical, but #86 should get merged before #86.

Reply:
Is #86 rebased on to of #85?, can we remove this so we can merge w/o blocking?